components/AnnotatedImage/AnnotatedImage.styles.ts

```diff
@@ -1,4 +1,4 @@
-export const root = 'relative self-start';
+export const root = 'relative self-start annotated-image';
```

Comment:
For if Forrest wants to track clicks for a specific group of components in Heap.